lib/ews/message_accessors.rb

```diff
@@ -43,37 +43,37 @@ module Viewpoint::EWS::MessageAccessors
   def send_message(opts = {}, &block)
     msg = Template::Message.new opts.clone
     yield msg if block_given?
-    if msg.has_attachments?
-      draft = msg.draft
-      msg.draft = true
-      resp = parse_create_item(ews.create_item(msg.to_ews))
-      msg.file_attachments.each do |attachment|
-        if attachment.kind_of?(Hash)
-          resp.file_attachment_from_hash(attachment)
-        elsif attachment.kind_of?(File) || attachment.kind_of?(Tempfile)
-          resp.file_attachment_from_file(attachment)
-        else
-          next
-        end
-      end
-      msg.inline_attachments.each do |attachment|
-        if attachment.kind_of?(Hash)
-          resp.inline_attachment_from_hash(attachment)
-        elsif attachment.kind_of?(File) || attachment.kind_of?(Tempfile)
-          resp.inline_attachment_from_file(attachment)
-        else
-          next
-        end
+    draft = msg.draft
```

Comment:
The condition was removed as EWS does not return message_id in response when simple sending is performed.